spec/controllers/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe V1::UsersController, type: :controller do
+
+  describe '#create' do
+    let(:params) do
+      {
+        user: {
+          email: 'user_one@exaple.com',
+          username: 'user',
+          password: 'password',
+          password_confirmation: 'password'
+        }
+      }
+    end
+
+    context 'user successful' do
+      it 'should create new user and return success' do
+        expect { post :create, params }.to change { User.count }.by(1)
+        expect(response.body).to include 'access_token'
```

Comment:
Oh, just realised you're possibly not using instance variables for these API requests...